roles/openshift_web_console/templates/console-smoke-test-pod.yaml.j2

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: openshift-web-console-smoke-test-pod
+  labels:
+    app: openshift-web-console-smoke-test
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/scheme: http
```

Comment:
I'd expect there to be a service, and we probably need https.

Reply:
You definitely need a secured service to expose the metrics endpoint; these annotations are not used in our monitoring configuration.